handlers/basic.py

```diff
@@ -23,10 +23,9 @@ def quip(update, context):
     update.message.reply_text(random.choice(values.quips))
 
 def get_group_id(update, context):
-    """Sends a quip everytime the command /quip is issued."""
+    """Replies with a group's chat ID"""
```

Comment:
Minor: "Returns" would make more sense here than "Replies"

Reply:
Would it? It's not really returning anything code-wise. It's replying to a user's command in the telegram chat. Maybe making it more descriptive, something link: `Replies to the user's message with the group's chat ID` would be better?